pyigm/abssys/utils.py

```diff
@@ -23,6 +23,7 @@
 from linetools.analysis import absline as ltaa
 from linetools.isgm.abscomponent import AbsComponent
 
+
 def chk_components(components, chk_match=False, chk_A_none=False, toler=0.2*u.arcsec):
     """ Performs checks on a list of components
```

Comment:
I'm not convinced that this checks on Components should be done here, as opposed to do these in linetools. I thought we agreed that AbsSystem should also be part of linetools (at least for now).